### ISSUE TITLE
Build on GHC 9

### ISSUE THREAD

**Issue by ocharles** [circuithub/rel8#31]
I don't think we can release without knowing we're compatible with the latest GHC

**Comment by ocharles** [comment]
As of 4d24729bd9b0ba735dc1985cf0f811a1031a2cab we can actually get to building Rel8, but it seems to hang on:

```
[ 10 of 118] Compiling Rel8.Schema.Nullability.Internal ( src/Rel8/Schema/Nullability/Internal.hs, dist/build/Rel8/Schema/Nullability/Internal.o, dist/build/Rel8/Schema/Nullability/Internal.dyn_o )
```

**Comment by ocharles** [comment]
Fixed in 84845f04843658d32e3d6d20e593cfaf37bd2ab2
